cypress/support/e2e.ts

```diff
@@ -47,12 +47,21 @@ beforeEach(() => {
     )
 
     if (Cypress.spec.name.includes('Premium')) {
-        cy.intercept('/api/users/@me/', { fixture: 'api/user-enterprise' })
+        cy.intercept('/api/users/@me/', { fixture: 'api/users/user-enterprise' })
```

Comment:
was this broken before because of this? I don't see that you moved any files 🤔

Reply:
I did move the file 🤔